src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -3414,7 +3445,7 @@ int MPL_gpu_fast_memcpy(void *src, MPL_pointer_attr_t * src_attr, void *dest,
     if (n == 1) {
         *(char *) d = *(char *) s;
     }
-#if defined(MPL_HAVE__MM256_STOREU_SI256) || defined(MPL_HAVE__MM512_STOREU_SI512)
+#if defined(MPL_HAVE__MM512_STOREU_SI512) || defined(MPL_HAVE__MM512_STREAM_SI512) || defined(MPL_HAVE__MM256_STREAM_SI256) || defined(MPL_HAVE__MM256_STOREU_SI256)
```

Comment:
I am wondering if the separate checks for `_mm512_stream_si512` and `_mm512_storeu_si512` are necessary. Is it possible for a compiler and architecture that support AVX512F (which we already checks) that does not have these two?

Reply:
I am not sure, autoconf tests both cases separately and define these two macro separately. So I think to be safe, they should be handled separately here as well.